components/gitops/staging/stone-stage-p01/gitops-service-postgres-rds-config-path.yaml

```diff
@@ -1,4 +1,4 @@
 ---
 - op: add
   path: /spec/dataFrom/0/extract/key
-  value: integrations-output/terraform-resources/app-sre-stage-01/stone-stage-p01/redhat-staging-gitopsvc-rds
+  value: integrations-output/terraform-resources/appsres05ue1/stone-stage-p01/redhat-staging-gitopsvc-rds
```

Comment:
it should be `stone-stage-p01-plnsvc-rds` instead of `redhat-staging-gitopsvc-rds`

Reply:
done